beacon_node/http_api/src/block_id.rs

```diff
@@ -4,7 +4,7 @@ use eth2::types::BlockId as CoreBlockId;
 use std::fmt;
 use std::str::FromStr;
 use std::sync::Arc;
-use types::{BlobsSidecar, Hash256, SignedBeaconBlock, SignedBlindedBeaconBlock, Slot};
+use types::{BlobSidecarList, Hash256, SignedBeaconBlock, SignedBlindedBeaconBlock, Slot};
```

Comment:
Actually we want `BlobSidecarList` to have each individual BlobSidecar arc'd right?

Reply:
or what Diva said above